src/main/java/uk/gov/justice/hmpps/prison/api/model/InmateDetail.java

```diff
@@ -201,6 +201,9 @@
     @ApiModelProperty(value = "current prison or outside with last movement information.", example="Outside - released from Leeds")
     private String locationDescription;
 
+    @ApiModelProperty(value = "Retricted patient details, if released to psychiatric hospital")
+    private RestrictivePatient restrictivePatient;
```

Comment:
I think we *HAVE* to have a huge disclaimer here, since it can imply that if this is not set they are not a restrictive patient - which right now is not true until each patient has their data fixed up.

Reply:
We will have to fix up the data definitely, but the last move is REL / HP then we can use that for now.